index.html

```diff
@@ -2800,7 +2800,7 @@ <h3>Concatenation of strings</h3>
 <aside class="example" title="Example of bidi generation guidance">
 	<p>Here is an example of a Note that you can include in your specification when recommending that implementations generate displayable [=natural language=] values:</p>
 	<div class="note" role="note" id="bidi-gen-note-example">
-	   <p class="example_note">When generating a value for <em><code>_field_name_</code></em>, [=producers=] need to ensure that directional changes <em>inside</em> the string will display correctly. When a [=producer=] assembles the value from multiple strings, individual substrings might need to be [=bidi isolated=] in order to prevent [=spillover=] effects. This can be done using markup (where available) or via the insertion of Unicode bidirectional controls. In addition, if the [=producer=] can supply the correct base direction as metadata, it allows the [=consumer=] to both isolate the string and set its direction, avoiding [=spillover=] effects between the full string and the context in which it is eventually displayed.  For more information see: <a href="https://www.w3.org/International/questions/qa-bidi-unicode-controls"><cite>How to use Unicode controls for bidi text</cite></a> and <a href="https://www.w3.org/TR/international-specs/#inline_changes">Inlines changes to base direction</a> [[INTERNATIONAL-SPECS]].</p>
+	   <p class="example_note">When generating a value for <em><code>_field_name_</code></em>, [=producers=] need to ensure that directional changes <em>inside</em> the string will display correctly. When a [=producer=] assembles the value from multiple strings, individual substrings might need to be [=bidi isolated=] in order to prevent [=spillover=] effects. This can be done using markup (where available) or via the insertion of Unicode bidirectional controls. In addition, if the [=producer=] can supply the correct base direction as metadata, it allows the [=consumer=] to both isolate the string and set its direction, avoiding [=spillover=] effects between the full string and the context in which it is eventually displayed.  For more information see: <a href="https://www.w3.org/International/questions/qa-bidi-unicode-controls"><cite>How to use Unicode controls for bidi text</cite></a> and <a href="https://www.w3.org/TR/international-specs/#inline_changes">Inline changes to base direction</a> [[INTERNATIONAL-SPECS]].</p>
```

Comment:
Since it's a section **within** specdev, we could use `[[[#inline_changes]]]` instead of `<a href="https://www.w3.org/TR/international-specs/#inline_changes">Inline changes to base direction</a>`, because:

1. The reader reading the editor's draft will not jump from ED to the /TR version.
2. The link will not fail to open when there is no internet (for example, I sometimes read specdev offline).
3. There is no need to enter the section title in the source text.